### PR TITLE
add satellite layer to edit sites map

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request -
 - add layer toggle on edit sites map
 - fly to user location(from config API) on register trees

### CODE REVIEW

**Comment by sarveshpro** [comment]
#779

**Comment by norbertschuler** [comment]
I added a Cloudflare worker to return the config for the development environment (https://app-development.plant-for-the-planet.org/public/v1.2/de/config and https://app-development.plant-for-the-planet.org/public/v1.2/en/config - no other languages) like for the production system with the location data included (CDN does not seem to work yet) in order to be able to test this. 

- [ ] First test seem to fail: 

Bildschirmfoto 2021-03-10 um 11 19 31

**Comment by norbertschuler** [comment]
- [ ] Not sure if this is related to this PR or a general bug, but the info icon from mapbox overlaps with the zoom controls.

Bildschirmfoto 2021-03-10 um 10 47 23

File: src/features/user/ManageProjects/components/ProjectSites.tsx [hunk 1]
```diff
@@ -487,9 +496,9 @@ function EditSite({ openModal, handleModalClose, changeSiteDetails, siteDetails,
     setGeoJson,
     geoJsonError,
     setGeoJsonError,
-    geoLocation:{
-      geoLatitude:36.96,
-      geoLongitude:-28.5
+    geoLocation: {
+      geoLatitude: 36.96,
```
Comment (norbertschuler) [inline src/features/user/ManageProjects/components/ProjectSites.tsx:500]: If you search for `36.96` you find a lot of these default values in our code. What is this place?

Comment (sarveshpro) [reply src/features/user/ManageProjects/components/ProjectSites.tsx:500]: That's the default map center I believe, which we use when we aren't looking at anything specific.

Comment (norbertschuler) [reply src/features/user/ManageProjects/components/ProjectSites.tsx:500]: Ok, but where does these coordinates come from - is it Atlantis? *just joking* 😁

**Comment by norbertschuler** [comment]
- [ ]  In my example project I had defined a polygon at a zoom level showing no satellite map when I switched to the satellite layer - but I assume we have to deal with this. After I zoomed out a little bit, the data got loaded
- [ ] Can we give the "Satellite" text a little padding to the left and right?
 
Bildschirmfoto 2021-03-10 um 13 16 26
Bildschirmfoto 2021-03-10 um 13 17 05

**Comment by norbertschuler** [comment]
- [ ] Just a question: Is this the zoom level we want to present the user for registering trees. I think we might zoom in more?

Bildschirmfoto 2021-03-10 um 13 15 13

**Comment by norbertschuler** [comment]
> * [x]  Just a question: Is this the zoom level we want to present the user for registering trees. I think we might zoom in more?

Bildschirmfoto 2021-03-12 um 19 03 21

That's probably a better zoom level to start. I recognized the Copyright remark and info icon are now completely deleted, are they? @sagararyal Is that ok not to have any reference at all?

**Review by norbertschuler — APPROVED** [review]
LGFM - please clarify if no map credentials need to be given any more. @sagararyal

**Pull request merged**